lib/SILOptimizer/IPO/GlobalOpt.cpp

```diff
@@ -300,7 +331,7 @@ static SILFunction *genGetterFromInit(StoreInst *Store,
   return GetterF;
 }
 
-/// If this is a read from a global let variable, map it.
+// If this is a write to a global let variable, map it.
```

Comment:
Is there some guiding principle for when you're using doc-comments vs. normal comments?

(I personally find the whole thing silly since no one would ever be reading a doc page to understand the internal implementation of a SIL pass, but if I know the rule I'll try to follow it)